Exercise_4/Adapters/PlaceHolderFragment.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using Android.App;
+using Android.OS;
+using Android.Views;
+using Android.Widget;
+using Newtonsoft.Json;
+using Fragment = Android.App.Fragment;
+
+namespace Exercise_4.Adapters
+{
+    public class PlaceHolderFragment : Fragment
+    {
+        [InjectView(Resource.Id.linearLayoutFragment)] private LinearLayout linearLayout;
+
+        [InjectView(Resource.Id.listView)] private ListView listView;
+
+        private List<string> datas = new List<string>();
```

Comment:
data is an uncountable noun.  Can not make it plural. You can name it `items` or `strings` because the data display is quite generic

Reply:
Done